src/gui/sdlmain.cpp

```diff
@@ -2877,11 +2877,9 @@ static SDL_Point parse_window_resolution_from_conf(const std::string &pref)
 		return {w, h};
 	}
 
-	LOG_WARNING("DISPLAY: Requested window resolution '%s' is not valid, "
-	            "falling back to '%dx%d' instead",
-	            pref.c_str(),
-	            FallbackWindowSize.x,
-	            FallbackWindowSize.y);
+	LOG_WARNING("DISPLAY: Invalid 'windowresolution' setting: '%s', "
+	            "using 'default'",
```

Comment:
Why not logging anymore what the default/fallback was? %dx%d is removed along with FallbackWindowSize.x,            FallbackWindowSize.y

Reply:
Found it unnecessary.